CommunicationManagementHelp.md

```diff
@@ -242,11 +242,14 @@ the recipient will need to log in using the email they specified in their [onboa
 To copy the sharable link:
 - Right-click on the desired file and select `Share` OR click on the desired file and click the share button near the top right of the screen
 - Click 'WHO HAS ACCESS' in the bottom left corner of the dialogue box that appears
+
+<img src="/images/who-has-access.PNG" alt="Link Sharing Off" width="300"/>
```

Comment:
This one is the one I noticed was out of date :)

Reply:
@ACharbonneau  Sorry what exactly is the change. I think the procedure you laid out is still relevant for copying links...

Reply:
so, yesterday when I went to get a link to share, the screen had changed to this: 
![image](https://user-images.githubusercontent.com/1719360/80809490-caf72480-8b8f-11ea-9477-c6b144448c1e.png)

did it not change for everyone?

Reply:
That's strange. But yes I still see the same images as those that exist in the repo already.

Reply:
huh. Ok. It must be some kind of phased rollout of new features thing? So then let's merge in these changes, and if/when the view changes for people who aren't me, we'll update then :)